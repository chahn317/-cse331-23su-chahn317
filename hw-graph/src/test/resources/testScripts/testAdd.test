# Tests addNode() and addEdge()

# Create a graph
CreateGraph graph1

# Add three nodes
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3

# Add self-edge
AddEdge graph1 n1 n1 e11

# Add more edges
AddEdge graph1 n1 n2 e12
AddEdge graph1 n1 n3 e13

# Add edge cycle
AddEdge graph1 n2 n3 e23
AddEdge graph1 n3 n1 e31
