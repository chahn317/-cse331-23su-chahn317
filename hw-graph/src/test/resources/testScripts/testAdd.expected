# Tests addNode() and addEdge()

# Create a graph
created graph graph1

# Add three nodes
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1

# Add self-edge
added edge e11 from n1 to n1 in graph1

# Add more edges
added edge e12 from n1 to n2 in graph1
added edge e13 from n1 to n3 in graph1

# Add edge cycle
added edge e23 from n2 to n3 in graph1
added edge e31 from n3 to n1 in graph1